cmd/sovereignnode/chainSimulator/tests/esdt/issue_test.go

```diff
@@ -0,0 +1,75 @@
+package esdt
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-go/config"
+	chainSim "github.com/multiversx/mx-chain-go/integrationTests/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components/api"
+	sovereignChainSimulator "github.com/multiversx/mx-chain-go/sovereignnode/chainSimulator"
+	"github.com/multiversx/mx-chain-go/vm"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	coreAPI "github.com/multiversx/mx-chain-core-go/data/api"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSovereignChain_Issue(t *testing.T) {
```

Comment:
perhaps?
```suggestion
func TestSovereignChain_IssueEsdt(t *testing.T) {
```

Reply:
changed to `_IssueFungible` to be more specific